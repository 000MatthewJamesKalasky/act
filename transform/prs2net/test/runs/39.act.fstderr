In expanding foo<> (inst: ct) (39.act:1)
In expanding -toplevel-
Channel: a
FATAL: Receive action in multiple concurrent blocks!
