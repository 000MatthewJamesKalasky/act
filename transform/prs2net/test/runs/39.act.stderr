In expanding foo<>
Channel: a
FATAL: Receive action in multiple concurrent blocks!
